### ISSUE TITLE
stage3 - improve description

### ISSUE THREAD

**Issue by RedJocker** [RedJocker/fixCinemaRoomManager#3]
description is not clear on how to calculate prices for rows

https://hyperskill.org/projects/198/stages/986/implement

**Comment by RedJocker** [comment]
proposed new description

Stage 3/5: stage3
Description

When you choose a ticket, you look not only at a particular seat but also it's cost. Let's implement the ability to check the ticket price.

Objectives

Show a ticket price, on a Toast, when you click on a seat.

Seat ticket price is the base price weighted by it's row position.

Seats closer to the screen have a fixed weight of 1.5 which is the max weight, the min weight is a target weight of 0.5. To calculate middle seats weights you should find the weight step, that is, the difference between max weight and min weight divided by the number of rows. The weight of each particular row decreases evenly starting from the max weight, using the weight step, the farther away you get from the screen.

The seat ticket price is calculated by multiplying the base ticket cost with the row weight
For example, if we have 10 rows with a base ticket cost of 10. The difference between max weight and min weight is 1.0. The weight step will be 0.1. Seat prices will be:

[(10 * 1.5), (10 * 1.4), (10 * 1.3), (10 * 1.2), (10 * 1.1), (10 * 1.0), (10 * 0.9), (10 * 0.8), (10 * 0.7), (10 * 0.6)]

Notice that the farthest away from screen row actual weight might be higher than the target min weight, but the closest to screen row actual weight is always equals to the max weight.

Examples
Example 1: an example of your application

[image]
 
The screenshot has been taken in the dark mode. If you prefer a light system mode, your application can use another color scheme.

```
<h2>Description</h2>

<p>When you choose a ticket, you look not only at a particular seat but also it's cost.
    Let's implement the ability to check the ticket price.
</p>



<h2>Objectives</h2>

<p>
    Show a ticket price, on a <code>Toast</code>, when you click on a seat.
</p>

<p>
    Seat ticket price is the base price weighted by it's row position.
</p>
<p>
    Seats closer to the screen have a fixed weight of <code>1.5</code> which is the max weight,
    the min weight is a target weight of <code>0.5</code>.
    To calculate middle seats weights you should find the weight step, that is,
    the difference between max weight and min weight divided by the number of rows.
    The weight of each particular row decreases evenly starting from the max weight, using the weight step,
    the farther away you get from the screen.
</p>

<p>
    The seat ticket price is calculated by multiplying the base ticket cost with the row weight
</p>

<p>
    For example, if we have <code>10</code> rows with a base ticket cost of <code>10</code>.
    The difference between max weight and min weight is <code>1.0</code>.
    The weight step will be <code>0.1</code>.
    Seat prices will be:
</p>

<p>
    <code>[(10 * 1.5), (10 * 1.4), (10 * 1.3), (10 * 1.2), (10 * 1.1), (10 * 1.0), (10 * 0.9), (10 * 0.8), (10 * 0.7), (10 * 0.6)]</code>
</p>

<p>
    Notice that the farthest away from screen row actual weight might be higher than the target min weight,
    but the closest to screen row actual weight is always equals to the max weight.
</p>



<h2>Examples</h2>

<p><strong>Example 1:</strong> <em>an example of your application</em></p>

<p><img alt="" height="452" src="https://ucarecdn.com/8007e418-eb82-4ad6-99f0-b5a6a26c8620/" width="214"></p>

<p>
    The screenshot has been taken in the dark mode.
    If you prefer a light system mode, your application can<em> </em>use another color scheme.
</p>

```

**Comment by zoyakostyreva** [comment]
Hi! I've checked your proposed descriptions and I like your suggestions. 

Do you think that a scheme illustrating these phrases will be useful?

> The seat ticket price is calculated by multiplying the base ticket cost with the row weight
> For example, if we have 10 rows with a base ticket cost of 10. The difference between max weight and min weight is 1.0. The weight step will be 0.1. Seat prices will be:
> 
> [(10 * 1.5), (10 * 1.4), (10 * 1.3), (10 * 1.2), (10 * 1.1), (10 * 1.0), (10 * 0.9), (10 * 0.8), (10 * 0.7), (10 * 0.6)]

I can ask our designer to create it.

**Comment by RedJocker** [comment]
Yes, that would be nice, if it's not too much work

**Comment by RedJocker** [comment]
after revision, with fixed tex code

```
<h2>Description</h2>

<p>When you choose a ticket, it&#39;s a good idea to consider its cost. Let&#39;s implement the ability to check the ticket price!</p>

<h2>Objectives</h2>

<ul>
	<li>Show a ticket price&nbsp;on the&nbsp;<code>Toast</code>&nbsp;when you click on a seat;</li>
	<li>The&nbsp;price is the base price weighted by its row position.&nbsp;Seats closer to the screen have a fixed maximum weight of&nbsp;<code>1.5</code>.&nbsp;The minimum weight is the target weight of <code>0.5</code>. To calculate the middle seat weights, find the weight step &mdash; the difference between the max and min weights divided by the number of rows. The weight of each particular row decreases evenly starting from the max weight to the min weight;</li>
	<li>The seat ticket price is calculated by multiplying&nbsp;the base ticket cost by&nbsp;row weight.</li>
</ul>

<p>For example, if we have <code>10</code> rows with a base ticket cost of <code>10</code>. The difference between the max and the min weight is <code>1.0</code>. The weight step is&nbsp;<code>0.1</code>. Seat prices are as follows:</p>

<p><span class="math-tex">\[[(10 \times 1.5), (10 \times 1.4), (10 \times 1.3), (10 \times 1.2), (10 \times 1.1), (10 \times 1.0), (10 \times 0.9), (10 \times 0.8), (10 \times 0.7), (10 \times 0.6)]\]</span></p>

<p>Mind that the farthest row&#39;s actual weight can be higher than the target minimum weight, but the closest row&#39;s actual weight is always equal to the maximum weight.</p>

<h2>Example</h2>

<p><strong>Example 1:</strong> <em>an example of your application</em></p>

<p><img alt="" height="452" src="https://ucarecdn.com/8007e418-eb82-4ad6-99f0-b5a6a26c8620/" width="214" /></p>
```

**Comment by RedJocker** [comment]
committed here, closing when updated on stepik
